dali/operators/displacement/rotate_params.h

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_DISPLACEMENT_ROTATE_PARAMS_H_
+#define DALI_OPERATORS_DISPLACEMENT_ROTATE_PARAMS_H_
+
+#include <sstream>
+#include <string>
+#include <vector>
+#include "dali/pipeline/operator/operator.h"
+#include "dali/kernels/imgproc/warp/affine.h"
+#include "dali/kernels/imgproc/warp/mapping_traits.h"
+#include "dali/kernels/imgproc/roi.h"
+#include "dali/operators/displacement/warp_param_provider.h"
+#include "dali/core/tensor_shape_print.h"
+#include "dali/core/format.h"
+
+namespace dali {
+namespace rotate_impl {
```

Comment:
```suggestion
namespace rotate { 
namespace impl {
```
maybe?

Reply:
A shy 'no'? I don't think that level of nesting is necessary.

Reply:
As a matter of fact, I'll remove this namespace and have the `using` directives in the function that needs them.